queries.sql

```diff
@@ -5,4 +5,30 @@ SELECT date_of_birth FROM animals WHERE NAME IN ( 'Agumon','Pikachu');
 SELECT name, escape_attempts FROM animals WHERE (weight_kg > 10.5);
 SELECT * FROM animals WHERE (neutered = true);
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE (weight_kg >= 10.4 AND weight_kg <= 17.3);
\ No newline at end of file
+SELECT * FROM animals WHERE (weight_kg >= 10.4 AND weight_kg <= 17.3);
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE Name like '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+UPDATE animals SET weight_kg = weight_kg * (-1);
+SAVEPOINT date_deletion
+UPDATE animals SET weight_kg = weight_kg * (-1) WHERE (weight_kg < 0);
+COMMIT;
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE (escape_attempts = 0);
+SELECT AVG(weight_kg)  FROM animals;
+SELECT neutered,AVG(escape_attempts) AS escape FROM animals GROUP BY (neutered) ORDER BY escape DESC LIMIT 1;
+SELECT MIN(weight_kg),MAX(weight_kg), species FROM animals GROUP BY species;
+SELECT species,AVG(escape_attempts) AS escape FROM animals WHERE date_of_birth BETWEEN  '1990-01-01' AND '2000-01-01' GROUP BY (species) ;
```

Comment:
- [ ] For your new screenshots after the second opinion, I still see the duplicate data - `Charmander` on most of your new screenshots, that should have been fixed before you begin to run your codes to generate new screenshots. In other words, I suggest that you clear your database, then insert the data again and continue with your query commands, then take screenshots. 
See the screenshots from your pull request after the second opinion, they are all muddled up. I also suggest that you try to use the query questions as labels for each screenshot to make it easy to know which screenshot is for which query question. 


![image](https://github.com/KAYEMBAJONATHAN/Vet_Clinic/assets/84156714/c690d40c-9561-46de-a912-c1c225c2cfa7)